Box.V2.Test.Integration/BoxSharedItemsManagerTestIntegration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Box.V2.Test.Integration
+{
+    [TestClass]
+    public class BoxSharedItemsManagerTestIntegration : BoxResourceManagerTestIntegration
+    {
+        [TestMethod]
+        public async Task SharedLink_LiveSession()
+        {
+            const string sharedLink = "https://app.box.com/s/70pecdxd6pvnd285rs4hqdp7zphgyqva";
+            const string password = "demo1234";
```

Comment:
not sure if it's a good idea to have the password (even this is a demo pwd of a shared item) in the repo?